src/components/MapView/Legends/ColorIndicator/index.tsx

```diff
@@ -11,13 +12,13 @@ const useStyles = makeStyles(() =>
       height: 10,
       width: 10,
       marginRight: 4,
-      backgroundColor: ({ color }: any) => color,
+      backgroundColor: ({ color, opacity }: any) => fade(color, opacity),
```

Comment:
While this was `any`before, we can definitely do better and give this a type :+1: